doc/i2p.md

```diff
@@ -133,3 +133,36 @@ listening port to 0 when listening for incoming I2P connections and advertises
 its own I2P address with port 0. Furthermore, it will not attempt to connect to
 I2P addresses with a non-zero port number because with SAM v3.1 the destination
 port (`TO_PORT`) is always set to 0 and is not in the control of Bitcoin Core.
+
+## i2pd
+
+### Bandwidth
+
+`i2pd` may route a large amount of general network traffic with default
+settings enabled. Check your routers configuration to limit the amount of this
+traffic that is being relayed.
+
+For `i2pd` this can be achieved by modifying the `bandwidth`, `share` and
+`transittunnels` options. For example to limit total I2P traffic to 256KB/s,
```

Comment:
```suggestion
`transittunnels` options in the `i2pd/contrib/i2pd.conf` file. For example, to limit total I2P traffic to 256KB/s,
```

Reply:
👍🏼 I did test that disabling the transit traffic still permitted connections to and from the SAM proxy, and it worked between my two nodes. I also double checked on libera-chat#i2pd-dev that this is intended behaviour, and it is. So if you want *only* bitcoin traffic on your I2P proxy, you can disable everything else for ultimate minimum bandwidth-usage!

Reply:
I think I might leave out the conf file path: 1) It might be in different locations for different people (e.g. `/etc/i2pd/i2pd.conf` or `/var/lib/i2pd/i2pd.conf` etc.) and 2) people might just be running with command line args.

Reply:
Suggestions

```diff
 ## Bandwidth
 
-I2P routers may route a large amount of general network traffic with default
-settings enabled. Check your router's configuration to limit the amount of this
-traffic that is being relayed.
-
-For `i2pd` controlling the amount of bandwidth being shared with the wider
-network can be achieved by modifying the `bandwidth`, `share` and
-`transittunnels` options. For example to limit total I2P traffic to 256KB/s,
-and share 50% of this limit for a maximum of 20 transit tunnels you can use:
+I2P routers may route a large amount of general network traffic with their
+default settings. Check your router's configuration to limit the amount of
+traffic relayed, if desired.
+
+With `i2pd`, the amount of bandwidth shared with the wider network can be
+adjusted with the `bandwidth`, `share` and `transittunnels` options in
+`i2pd.conf`. For example, to limit I2P traffic to 256KB/s and share 50% of this
+limit for a maximum of 20 transit tunnels:
```

Reply:
> I think I might leave out the conf file path: 1) It might be in different locations for different people (e.g. `/etc/i2pd/i2pd.conf` or `/var/lib/i2pd/i2pd.conf` etc.) and 2) people might just be running with command line args.

Agree WRT the path. Maybe give the conf filename for both `i2pd` and `i2p router` or for neither.

Reply:
I believe this is addressed [here](https://github.com/bitcoin/bitcoin/pull/25993/commits/27b9e098d163c292753cdd3df9cefefa28b648f8#diff-629b3acb5e1cd9ad21f4f55bc95822e79f22f15da47f5de82284c15cbe8bb47bR160-R161) in response to vasild's comment.

Reply:
> I believe this is addressed [here](https://github.com/bitcoin/bitcoin/pull/25993/commits/27b9e098d163c292753cdd3df9cefefa28b648f8#diff-629b3acb5e1cd9ad21f4f55bc95822e79f22f15da47f5de82284c15cbe8bb47bR160-R161) in response to vasild's comment.

How?

Mind having a look at https://github.com/bitcoin/bitcoin/pull/25993#discussion_r968230102? It would fix a few punctuation issues and improve clarity/brevity.

Reply:
Ok I have adopted the changes from https://github.com/bitcoin/bitcoin/pull/25993#discussion_r968230102

> How?

As the final clause now reads:

```
Similar bandwidth configuration options in the Java I2P router can be found in
the `http://127.0.0.1:7657/config`, under the "Bandwidth" tab.
```

Which I think, is roughly the equivalent of the `i2p router` conf file?